src/transformers/modeling_mobilebert.py

```diff
@@ -0,0 +1,1054 @@
+# MIT License
+#
+# Copyright (c) 2020 lonePatient
```

Comment:
```suggestion
# Copyright (c) 2020  The Google AI Language Team Authors, The HuggingFace Inc. team and github/lonePatient
```

I think this would be the right copyright, right @thomwolf ?